test/DomainRegistry.test.js

```diff
@@ -0,0 +1,99 @@
+const { expect } = require("chai");
+
+describe("DomainRegistry", function () {
+  let domainRegistry; // Объявляем переменную для контракта
+  let tldToRegister; // Объявляем переменную для домена
+  let owner; // Объявляем переменную для аккаунта владельца
```

Comment:
Do you really need to comment what is obvious? If you work with ChatGPT that does not mean you should just copy-paste.
 -5

Reply:
all comments removed